uaclient/entitlements/cis.py

```diff
@@ -8,4 +8,7 @@ class CISEntitlement(repo.RepoEntitlement):
     title = "CIS Audit"
     description = "Center for Internet Security Audit Tools"
     repo_key_file = "ubuntu-securitybenchmarks-keyring.gpg"
-    packages = ["ubuntu-cisbenchmark-16.04"]
+
+    @property
+    def packages(self):
+        return ["ubuntu-cisbenchmark-16.04"] + super().packages
```

Comment:
You should drop this method altogether as we should get it from the resource's additionalPackages directive (and looks like an update in the cis-audit resource definition changed the package name anyway to usg-*)  https://github.com/CanonicalLtd/ua-contracts/blob/develop/products/resource-cis.yaml